var/spack/repos/builtin/packages/zoltan/package.py

```diff
@@ -85,6 +85,8 @@ def install(self, spec, prefix):
             config_cflags.append(self.compiler.pic_flag)
             if spec.satisfies('%gcc'):
                 config_args.append('--with-libs={0}'.format('-lgfortran'))
+            if spec.satisfies('%intel'):
+                config_args.append('--with-libs={0}'.format('-lifcore'))
```

Comment:
why does this use format at all? Why not directly use `'--with-libs=-lifcore'`?

Reply:
good point; I just followed the already existing %gcc style ...